pkg/menu/menu.go

```diff
@@ -16,12 +16,12 @@ package menu
 
 import (
 	"io/ioutil"
-	"log"
 
 	"github.com/onosproject/onos-docs/pkg/common"
 	"github.com/onosproject/onos-docs/pkg/manifest"
 	"github.com/onosproject/onos-docs/pkg/types"
 	utils "github.com/onosproject/onos-docs/pkg/utils"
+	"k8s.io/klog"
```

Comment:
To be consistent with the rest of the projects I’d put this with an alias ‘log “k8s.io/klog”’

Reply:
Done